src/contracts/misc/GhoCcipSteward.sol

```diff
@@ -130,7 +130,13 @@ contract GhoCcipSteward is RiskCouncilControlled, IGhoCcipSteward {
   }
 
   /// @inheritdoc IGhoCcipSteward
-  function RISK_COUNCIL() public view override returns (address) {
+  function getTimeLockState() external view override returns (uint40, uint40) {
```

Comment:
This feels a bit more aligned with the other contracts
```suggestion
  function getCcipTimelocks() external view override returns (uint40, uint40) {
```

Reply:
[5c7abac](https://github.com/aave/gho-core/pull/438/commits/5c7abacde5af9ad32b525c978a3bb95a0d395e99)